test/games/test_log_file_export.py

```diff
@@ -11,29 +12,34 @@
 
 if typing.TYPE_CHECKING:
     import _pytest.python
-    import pytest
-    import pytest_mock
     from conftest import TestFilesDir
 
+    from randovania.layout.base.cosmetic_patches import BaseCosmeticPatches
+
+
+def _update_committed(data_path: Path, cosmetic_path: Path, data: dict, cosmetic_patches: BaseCosmeticPatches) -> None:
+    """
+    Updates the files that are used as reference value for later runs. Make sure to always keep this call commented out.
+    """
+    json_lib.write_path(data_path, data)
+
+    if cosmetic_patches != type(cosmetic_patches)():
+        # If not the default
+        json_lib.write_path(cosmetic_path, cosmetic_patches.as_json)
+    else:
+        cosmetic_path.unlink(missing_ok=True)
+
+    raise NotImplementedError("This function should not be called in normal execution.")
```

Comment:
Not implemented error is not really what I would expect here.

Reply:
I dunno what's best to actually use here. Creating a new type if pretty overkill